docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,33 @@
+# Proposal for FederatedCatalog with Tractus-X distribution
+
+## Decision
+
+The **proposed solution is to have a Federated Catalog as own runnable service** capable of crawling all the wanted catalogs and expose that data.
+
+
+## Rationale
+
+The Federated Catalog aims at providing the aggregation of catalogs from multiple participants in a centralized point to reduce latency.
+
+Choosing a solution that decouples from Control Plane (like the one used for the Data Plane) and able to be scalable will future-proof the Federated Catalog as a feature and embraces wider usage. 
+Like so, having a Federated Catalog service, scalable on its own based on user needs, and not on Control Plane, allows the best resource management in the long term. It also enables adding features to the Federated Catalog without the need to change the Control Plane logic (assuming no API usage change).
+
+Additionally, the Federated Catalog deployment flow can be very similar to EDC components, ensuring the consistency and no need for additional know-how to handle the CI.
+
+Overall, aiming at independent upgrades and instance scalability, the decision above is more suitable when compared with the Federated Catalog embedded and shipped as any other extension in the ControlPlane.
+
+
+
+## Approach
+
+The Federated Catalog running as own instance can be performed either by running respective Docker image in a container or running a generated `jar`. Both approaches should be very straightforward 
+
+> In case of aiming at testing locally, the [Readme.md](https://github.com/eclipse-edc/FederatedCatalog/blob/main/README.md) includes a very helpful guide for both approaches mentioned. 
```

Comment:
The idea was to show the possibility of running a docker image (running the jar underneath) and the option of the user running the jar with needed vars without docker. Both cases, as expected, would be the same jar running. That said, you are right, need to change to a something more correct.

> As Tractus-X EDC project I guess this will result in publishing the federated catalog image and adding it to the Helm Charts

yes, this is the goal here, similar to other existing examples. Will not add here since what you commented here should already cover this idea.

Changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/a452459467e7d78d60ce263a2fecf6ea2c14c205).